.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,13 +13,8 @@ Fixes #[issue_no]
 ## Checklist before merge
 
 **Developer's responsibilities**
-* [ ] Approved by at least one developer: *enter reviewer here*
+* [ ] Approved by at least one developer
```

Comment:
I'd prefer if it was written:
- [ ] Assign one or two developers
- [ ] Change code if reviewer(s) has/have requested it

Because we agreed that OP decides how many reviews are needed, this should be a todo for the OP.

Reply:
Would appreciate this too